src/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars-box {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+}
+
+.stars-box__stars {
+  display: flex;
+}
+
+.stars-box__rewies {
+  align-self: center;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.stars-box__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin: 0 4px 0 0;
+}
+
+.stars-box__stars > .stars-box__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
check please this part. accordingly to your task with stars. it's about to include modifier in html. and here use part from css wich includes choosing the number  with nth-child selectors. main idea IMHO is: change the number in the end of modifier in html and get proper amount of Active Stars.